pkg/js/README.md

```diff
@@ -23,7 +35,7 @@ import 'package:js/js.dart';
 external String stringify(Object obj);
 ```
 
-#### Classes and Namespaces
+### Using namespaces and defining classes
```

Comment:
I'm not sure if we want to call this "defining classes". The `class` we write here is a representation of the interface of a JS class which is defined elsewhere.

Reply:
Maybe: Using JavaScript namespaces and classes
?